contributing/code/tests.rst

```diff
@@ -51,8 +51,9 @@ what's going on and if the tests are broken because of the new code.
 
 .. tip::
 
-    On Windows, install the `ConEmu`_ or `ANSICON`_ free applications to see
-    colored test results.
+    On Windows, install the `ConEmu`_, `ANSICON`_ or `Mintty`_ free applications
+    to see colored test results.
```

Comment:
hmm, this sentence is actually related to the PHPUnit color support.  This means it depends on https://github.com/sebastianbergmann/environment/pull/11 rather than on the Symfony PR (PHPUnit does not use the Console component)